File: src/types/property.ts
export type Property = {
  id: number;
  title: string;
  area: string;
  address: string;
  price: number;
  sqm: number;
  lat: number;
  lng: number;
  createdAt: string;
  isOutlier?: boolean;
};

export type PropertyFilters = {
  minPrice?: number;
  maxPrice?: number;
  minSqm?: number;
  maxSqm?: number;
  area?: string;
};

File: GeoDataProcessingTool/Models/Property.cs
using System;

namespace GeoDataProcessingTool.Models;

public class Property
{
    public int Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Area { get; set; } = string.Empty;
    public string Address { get; set; } = string.Empty;
    public double Price { get; set; }
    public double Sqm { get; set; }
    public double Lat { get; set; }
    public double Lng { get; set; }
    public string CreatedAt { get; set; } = string.Empty;
    public bool IsOutlier { get; set; }
}
